### DISCUSSION TITLE
Enable Zperf tool for stm32h7 or stm32u5 series of controller

### DISCUSSION THREAD

**Discussion by SadanandPenore-eaton** [zephyrproject-rtos/zephyr#67016] [General] [answered]
Hello Zephyr team,
We want to enable the zperf tool for stm32h7 or stm32u5 controller.
As per documents, above boards are not supported. https://docs.zephyrproject.org/latest/samples/net/zperf/README.html
Can you please let us know the guideline for making it work on stm32h7 or u5 series of controller.

**Comment by KozhinovAlexander** [comment, marked as answer]
Just used to complie it on my nucleo_h745zi_q and it can be executed. Also `zperf help` works. I am not sure if performance tests will be running as suggested, need more investigation here.
In general - the list of supported boards means - it was tested with this boards. Usually one have to complie and test it on own board too :)

**Comment by jukkar** [comment]
As a general rule, the networking stack and applications is made to be board agnostic (if possible). So, as already mentioned by @KozhinovAlexander, if the documentation says "Supported boards", it means that it was at least tested with the said boards but it will probably work with other boards too. We are just not able to test with all boards, there are so many of those. Only if the documentation says that some board is not supported, then that board will not work with the sample or code.
For the zperf sample, as long as the board has network connectivity and enough memory, it should work fine.
